docs/data/base/components/input/UseInput.tsx

```diff
@@ -50,11 +51,14 @@ const CustomInput = React.forwardRef(function CustomInput(
   props: React.InputHTMLAttributes<HTMLInputElement>,
   ref: React.ForwardedRef<HTMLInputElement>,
 ) {
-  const { getRootProps, getInputProps } = useInput(props, ref);
+  const { getRootProps, getInputProps } = useInput(props);
+
```

Comment:
```suggestion

// Make sure that both the user's ref and the hook refs are applied on the input element
```

Or something similar.